effekt/jvm/src/main/scala/effekt/Driver.scala

```diff
@@ -35,64 +35,80 @@ trait Driver extends kiama.util.Compiler[EffektConfig, EffektError] { outer =>
    *
    * In LSP mode: invoked for each file opened in an editor
    */
-  override def compileSource(source: Source, config: EffektConfig): Unit = try {
-    val src = if (source.name.endsWith(".md")) { MarkdownSource(source) } else { source }
+  override def compileSource(source: Source, config: EffektConfig): Unit = {
+    val src = if (source.name.endsWith(".md")) {
+      MarkdownSource(source)
+    } else {
+      source
+    }
```

Comment:
This confused me at first since I couldn't tell what changed -- only the `try` got moved, right?

Reply:
It is hard to see indeed; indenting the code triggered a reformat by intelliJ. If you want I can try to use another tool to reproduce the change with smaller diff.
What changed is: `src` is being initialized before the `try` block, the statements in finally use `src` rather than `source`.